sycl/source/detail/queue_impl.cpp

```diff
@@ -330,6 +344,48 @@ void queue_impl::addSharedEvent(const event &Event) {
   MEventsShared.push_back(Event);
 }
 
+event queue_impl::submit_impl(const std::function<void(handler &)> &CGF,
+                              const std::shared_ptr<queue_impl> &Self,
+                              const std::shared_ptr<queue_impl> &PrimaryQueue,
+                              const std::shared_ptr<queue_impl> &SecondaryQueue,
+                              const detail::code_location &Loc,
+                              const SubmitPostProcessF *PostProcess) {
+  // Flag used to detect nested calls to submit and report an error.
```

Comment:
This comment looks redundant now.

Reply:
good catch, thanks

Reply:
removed in https://github.com/intel/llvm/pull/13659/commits/ec7c6b013e6e822a7d9319d2cc13b48862461370